src/main/java/net/imagej/ops/topology/TopologyNamespace.java

```diff
@@ -3,6 +3,7 @@
 import net.imagej.ops.AbstractNamespace;
 import net.imagej.ops.Namespace;
 import net.imagej.ops.OpMethod;
+import net.imagej.ops.topology.eulerCharacteristic.EulerCorrection;
```

Comment:
Question for you: are there Euler-related ops that are not "Euler characteristics"? Because if not, I would vote for `topology.euler` over `topology.eulerCharacteristic`.

Actually though, both of these packages violate the naming convention. From the [Naming guide](https://github.com/imagej/imagej-ops/wiki/Naming):
> Each kind of operation lives in an associated package matching its namespace and op name. For example, ops named math.add are declared in the net.imagej.ops.math.add package.